src/QMCWaveFunctions/tests/test_einset_spinor.cpp

```diff
@@ -11,6 +11,7 @@
 //////////////////////////////////////////////////////////////////////////////////////
 
 
+#include "QMCTools/QMCFiniteSize/FSUtilities.h"
```

Comment:
QMCWaveFunctions is at a lower level than QMCTools. Cannot have this include.

Reply:
Don't know how that ended up there

Reply:
fixed